test/conformance/chainsaw/generate/clusterpolicy/standard/data/nosync/generate-on-eviction-subresource-trigger/chainsaw-test.yaml

```diff
@@ -0,0 +1,99 @@
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  name: generate-on-eviction-subresource-trigger
+spec:
+  steps:
+  - name: create policy
+    use:
+      template: ../../../../../../_step-templates/create-policy.yaml
+      with:
+        bindings:
+        - name: file
+          value: policy.yaml
+  - name: step-01
+    try:
+    - apply:
+        file: namespace.yaml
+  - name: wait policy ready
+    use:
+      template: ../../../../../../_step-templates/cluster-policy-ready.yaml
+      with:
+        bindings:
+        - name: name
+          value: zk-kafka-address
+  - try:
+    - assert:
+        file: namespace-ready.yaml
+  - name: step-02
+    try:
+    - command:
+        args:
+        - run
+        - nginx
+        - --image=nginx
+        - -n
+        - test-generate-eviction
+        entrypoint: kubectl
+    - command:
+        args:
+        - wait
+        - --for=condition=Ready
+        - pod/nginx
+        - -n
+        - test-generate-eviction
+        entrypoint: kubectl
+    - command:
+        args:
+        - evict
+        - nginx
+        - -n
+        - test-generate-eviction
+        entrypoint: kubectl
```

Comment:
Is there an "evict" sub-command for "kubectl"? 

https://github.com/kyverno/kyverno/actions/runs/11863342597/job/33120537741?pr=11594#step:3:2193

Reply:
added this plugin to support this : https://github.com/kyverno/kyverno/pull/11594/files#diff-701db5e555679ac7a6a0a80d67b00395c23aa6cf08ce4e7af083f516e534f2cfR184 . Seems like there is issue with it. Do you what is the problem here?

Reply:
Hopefully the new commit should fix it : https://github.com/kyverno/kyverno/pull/11594/files#diff-701db5e555679ac7a6a0a80d67b00395c23aa6cf08ce4e7af083f516e534f2cfR185